### PR TITLE
Included section on ML REST API task

### PR DESCRIPTION
**Pull request by starlord-daniel**
To provide documentation of the commonly used ML REST API task (e.g. by the [MLOpsPython template](https://github.com/microsoft/MLOpsPython)), this section is introduced.

### CODE REVIEW

**Comment by PRMerger20** [comment]
@starlord-daniel : Thanks for your contribution! The author(s) have been notified to review your proposed change. @azooinmyluggage

**Comment by ktoliver** [comment]
@NilsPohlmann Could you review the proposed update to your article?

Thanks.

**Review by steved0x — APPROVED** [review]

**Comment by steved0x** [comment]
#sign-off

**Comment by PRMerger17** [comment]
@steved0x: I'm sorry - only the author of this article, @azooinmyluggage, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

File: docs/pipelines/targets/azure-machine-learning.md [hunk 1]
```diff
@@ -148,6 +148,29 @@ Depending on what "best" looks like for your scenario, you may need to create a
...
+  displayName: 'Submit AML pipeline'
+  inputs:
+    azureSubscription: '$(SERVICE_CONNECTION)'
+    PipelineId: '$(PIPELINE_ID)'
+    ExperimentName: '$(EXPERIMENT_NAME)'
+    PipelineParameters: '"ParameterAssignments": {"parameter-example": 1 }'
+```
+
+This is a AML Workspace REST API task to invoke ML pipelines, scripts and notebooks.
+A similar behavior to this task can be achieved by invoking the following [Azure ML CLI command](https://docs.microsoft.com/en-us/cli/azure/ml(v1)/run?view=azure-cli-latest#az_ml_v1__run_submit_pipeline).
```
Comment (GitHubber17) [inline docs/pipelines/targets/azure-machine-learning.md:167]:
```suggestion
A similar behavior to this task can be achieved by invoking the following [Azure ML CLI command](https://docs.microsoft.com/cli/azure/ml(v1)/run?view=azure-cli-latest#az_ml_v1__run_submit_pipeline).
```

File: docs/pipelines/targets/azure-machine-learning.md [hunk 2]
```diff
@@ -148,6 +148,29 @@ Depending on what "best" looks like for your scenario, you may need to create a
...
+
+The additional benefit of the az ml task, besides submitting the pipeline with all specified PipelineParameters is, that it represents for the AML pipeline run status in the Azure Pipeline and doesn't need an unspecified amount of wait time for the task to finish.
```
Comment (GitHubber17) [inline docs/pipelines/targets/azure-machine-learning.md:169]:
```suggestion
The additional benefit of the az ml task, besides submitting the pipeline with all specified PipelineParameters, is that it represents for the AML pipeline run status in the Azure Pipeline and doesn't need an unspecified amount of wait time for the task to finish.
```

**Comment by PRMerger3** [comment]
@GitHubber17 : Thanks for your contribution! The author(s) have been notified to review your proposed change. @azooinmyluggage

**Pull request merged**